mopidy/models/storage.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import, unicode_literals
+
+import gzip
+import json
+import logging
+import os
+import tempfile
+
+import mopidy
+from mopidy import models
+from mopidy.internal import encoding
+
+logger = logging.getLogger(__name__)
+
+
+def load(path):
+    """
+    Deserialize data from file.
+
+    :param path: full path to import file
+    :type path: str
+    :return: deserialized data
+    :rtype: dict
+    """
+    if not os.path.isfile(path):
+        logger.info('File does not exist: %s.', path)
+        return {}
+    try:
+        with gzip.open(path, 'rb') as fp:
+            return json.load(fp, object_hook=models.model_json_decoder)
+    except (IOError, ValueError) as error:
+        logger.warning(
+            'Loading JSON failed: %s',
+            encoding.locale_decode(error))
+        return {}
```

Comment:
I guess the proper way to expose this would be to have it raise and exception. That way we always return a dict or explode. Which is a bit more predictable. Other question is of course if we need to be able to distinguish this.

How about just putting in a `TODO` question about this and postponing it.